dockerbuild/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM archlinux
+LABEL maintainer="github@danman.eu"
+
+ENV GCC=nds32le-elf-gcc-9.3.0-1-x86_64.pkg.tar.xz
+ENV NEWLIB=nds32le-elf-newlib-3.3.0-1-any.pkg.tar.xz
+
+RUN curl https://github.com/kittennbfive/lkv373-tools/releases/download/0.01/$GCC -L -O
+RUN curl https://github.com/kittennbfive/lkv373-tools/releases/download/0.01/$NEWLIB -L -O
+
+RUN pacman -Sy --noconfirm nds32le-elf-binutils
```

Comment:
As I said, this needs the `u` option as we need to do a system upgrade. Otherwise, we will be doing a partial update, which is not supported. `nds32le-elf-binutils` doesn't need to be explicitly installed, but if you want to, you can save time by adding the `--needed` option.

Reply:
I don't want to upgrade inside, instead I want to build always from the latest image.
What do you mean by "unsupported"? 
Have you tested this?

Reply:
Why don't you want to update? If you want to install something from the repos, you must do a system upgrade, if you don't you might run into issues.

See https://wiki.archlinux.org/index.php/System_maintenance#Partial_upgrades_are_unsupported